File: Udemy/Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaSequencial/src/application/ProblemaTroco.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaTroco {

	public static void main(String[] args) {
		/*
		 * Problema "troco"
Fazer um programa para calcular o troco no processo de pagamento de um produto de uma mercearia.
O programa deve ler o preço unitário do produto, a quantidade de unidades compradas deste produto,
e o valor em dinheiro dado pelo cliente (suponha que haja dinheiro suficiente). Seu programa deve
mostrar o valor do troco a ser devolvido ao cliente.
		 * */
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double preco, dinheiro, troco;
		int qtd;
		
		System.out.print("Preço unitário do produto: ");
		preco = sc.nextDouble();
		System.out.print("Quantidade comprada: ");
		qtd = sc.nextInt();
		System.out.print("Dinheiro recebido: ");
		dinheiro = sc.nextDouble();
		
		troco = dinheiro - preco * qtd;
		System.out.println("TROCO = "+String.format("%.2f", troco));
		
		sc.close();
	}

}

File: Algoritmos e Logica de Programacao/C++/Estrutura Sequencial/ProblemaTroco.cpp
/*
Problema "troco"
Fazer um programa para calcular o troco no processo de pagamento de um produto de uma mercearia.
O programa deve ler o preço unitário do produto, a quantidade de unidades compradas deste produto,
e o valor em dinheiro dado pelo cliente (suponha que haja dinheiro suficiente). Seu programa deve
mostrar o valor do troco a ser devolvido ao cliente.
*/

#include <iostream>
#include <iomanip>

using namespace std;

int main(){
	
	double preco, dinheiro, troco;
	int qtd;
	
	cout << "Preço unitário do produto: ";
	cin >> preco;
	cout << "Quantidade comprada: ";
	cin >> qtd;
	cout << "Dinheiro recebido: ";
	cin >> dinheiro;
	cout << fixed << setprecision(2);
	troco = dinheiro - (preco * qtd);
	cout << "TROCO = " << troco;

    return 0;
}
